### PR TITLE
Road to v0.3

### PR DESCRIPTION
**Pull request by fedebenelli**
- [X] UNIFAC
- [x] $G^E$ based mixing rules
- [x] Parameter fitting procedures

### CODE REVIEW

**Comment by fedebenelli** [comment]
Michelsen Modified-Huron-Vidal (MHV1) Added as a first example of  a $G^E$-based mixing rule

**Comment by fedebenelli** [comment]
While this won't be the last iteration, a basic parameter fitting logic has been added. This probably would be enough for releasing v0.3.

Opinions? @SalvadorBrandolin

File: app/isotherm.f90
Comment (SalvadorBrandolin) [inline app/isotherm.f90:1]: sus

**Review by SalvadorBrandolin — APPROVED** [review]
mucho texto (⌐ ͡■ ͜ʖ ͡■)

**Pull request merged**